contracts/src/TokenRemote/ERC20TokenRemote.sol

```diff
@@ -35,14 +58,33 @@ contract ERC20TokenRemote is IERC20TokenTransferrer, ERC20, TokenRemote {
      * @param tokenSymbol The symbol of the ERC20 token.
      * @param tokenDecimals_ The number of decimals for the ERC20 token.
      */
-    constructor(
+    function initialize(
         TokenRemoteSettings memory settings,
         string memory tokenName,
         string memory tokenSymbol,
         uint8 tokenDecimals_
-    ) TokenRemote(settings, 0, tokenDecimals_) ERC20(tokenName, tokenSymbol) {
-        _decimals = tokenDecimals;
+    ) public initializer {
+        __ERC20TokenRemote_init(settings, tokenName, tokenSymbol, tokenDecimals_);
```

Comment:
Should we remove the trailing _ from `tokenDecimals_` now that the storage slot variable has a leading _?